### ISSUE TITLE
max-len config not working v3.9.1

### ISSUE THREAD

**Issue by vkbansal** [eslint/eslint#7556]
* **ESLint Version:** 3.9.1
* **Node Version:** 7.0.0
* **npm Version:** 3.10.8

**What parser (default, Babel-ESLint, etc.) are you using?**
Babel-ESlint

```
rules:
    max-len:
        - 2
        - 120
        - code: 120
           ignoreString: true
           ignoreUrls: true
           ignoreTemplateLiterals: true
```

**What did you expect to happen?**
The lint should run

**What actually happened? Please include the actual, raw output from ESLint.**
```
/Users/vivek.bansal/Projects/my-project/.eslintrc.yml:
	Configuration for rule "max-len" is invalid:
	Value "[object Object]" no schemas match.
```

If I remove options, it works
```
rules:
    max-len:
        - 2
        - 120
```

**Comment by platinumazure** [comment]
According to the [docs](http://eslint.org/docs/rules/max-len#options), you can use either a numeric option or an object option, not both. The `code` object key is the same as the numeric option. So basically, you just need to remove the numeric option in your case. Hope this helps!

**Comment by vkbansal** [comment]
```
    max-len:
        - 2
        - code: 120
          ignoreString: true
          ignoreUrls: true
          ignoreTemplateLiterals: true
```

This also gives same error

**Comment by not-an-aardvark** [comment]
Your config has `ignoreString`, which isn't a valid option name. I think you meant to use `ignoreStrings` instead.

**Comment by vkbansal** [comment]
Yes you are right.

The error message should have been more descriptive

**Comment by not-an-aardvark** [comment]
I'm going to close this issue since it looks like the issue was an invalid schema, but feel free to continue the discussion.

**Comment by ejosafat** [comment]
I got the same problem, using eslint 3.0.1, if I use 
```
'max-len': { code: 120, ignoreUrls: true, ignoreStrings: true, ignoreTemplateLiterals: true }
```
I get this error:
Severity should be one of the following: 0 = off, 1 = warn, 2 = error (you passed '{ code: 120,  ignoreUrls: true,  ignoreStrings: true,  ignoreTemplateLiterals: true }').

If I use 
```
[2, { code: 120, ignoreUrls: true, ignoreStrings: true, ignoreTemplateLiterals: true }]
```
I get
Value "[object Object]" no schemas match.

I get that error using "error" instead of 2 as well.

I also tried
```
['error', 120, 'ignoreUrls', 'ignoreStrings', 'ignoreTemplateLiterals' ]
```

but get an error as well, don't know what else to try and I haven't found any real example of how to use this rule properly in the docs. Any help would be appreciated.
